### ISSUE TITLE
localization failed

### ISSUE THREAD

**Issue by LevGlukhov** [xdan/datetimepicker#331]
In current version localization doesn't work properly. Try to switch language in settings as it described in documentation in param - lang. But only english language displayed in calendar. Other params work normally.

**Comment by extremumratio** [comment]
Привет русскоговорящим!
'lang' was removed at my suggestion because it is impossible to set the language setting for one widget. See https://github.com/xdan/datetimepicker/pull/323

**Comment by extremumratio** [comment]
Yes, this is not in the documentation. Sorry... It makes me sad too. Did I or someone else will make Github documentation.

**Comment by extremumratio** [comment]
Or we can ask the author to update the documents on its website. )
